### ISSUE TITLE
Heavily reduce the size of the Wither Storm's top when hit with the Formidibomb

### ISSUE THREAD

**Issue by GearsawStudios** [nonamecrackers2/crackers-wither-storm-mod#836]
When the Wither Storm is hit with the Formidibomb, it does lose a large amount of mass from the top layer to account for the mini storms that it makes. However, it also makes it look like it just shrugged off the blast and just gets back up. Even though it quickly repairs itself because all the matter is on the ground, it still doesn't make sense that phase six is almost the same size. Perhaps changing the model of phase six to be closer in size to the first half of phase five would make it seem like to took a large amount of damage. Perhaps for the extra details some flesh clumps reused from when it dies for real could fall out only to be reabsorbed to account to account for the missing mass.

TL;DR Give phase 6 Wither Storm (split, three heads) a smaller model to account for the formidibomb's damage, adding flesh clumps behind it to be reabsorbed to account for how to repairs so quick.

**Comment by nonamecrackers2** [comment]
As stated in another github issue, updating/changing the models isn't in my best interest as of now. I do like this idea though and will keep it open for future reference.

**Comment by nonamecrackers2** [comment]
Closing since this goes against the new guidelines (drastic model changes)
